examples/credential-registry/src/lib.rs

```diff
@@ -0,0 +1,1969 @@
+//! This smart contract implements an example on-chain registry for the public
+//! part of verifiable credentials (VCs).
+//!
+//! # Description
+//!
+//! The contract keeps track of credential public data, allows for managing the
+//! VC life cycle and querying VCs data and status. The intended users are
+//! issuers of VCs, holders of VCs, revocation authorities and verifiers.
+//!
+//! ## Issuer's  functionality
+//!
+//! - register a new credential;
+//! - revoke a credential;
+//! - restore (cancel revocation of) a revoked credential;
+//! - register/remove revocation authority keys;
+//! - update the issuer's metadata;
+//! - update the credential metadata;
+//! - add/update credential types.
+//!
+//! Revocation authorities are some entities chosen by the issuer that have
+//! revocation capabilities. Their public keys are registered by the issuer and
+//! a revocation authority signs a revocation message with the corresponding
+//! private key.
+//!
+//! ## Holder's functionality
+//!
+//! - revoke a credential by signing a revocation message.
+//!
+//! Revocation authority's functionality
+//!
+//! - revoke a credential by signing a revocation message.
+//!
+//! ## Verifier's functionality
+//!
+//! - view credential status to verify VC validity;
+//! - view credential data to verify proofs (verifiable presentations) requested
+//!   from holders.
+
+// TODO: remove the CIS-2 dependency once MetadataUrl is moved to concordium-std
```

Comment:
Please remove this TODO once done.,

Reply:
Fixed, Addressed in https://github.com/Concordium/concordium-rust-smart-contracts/pull/278 merged.